iceoryx_posh/include/iceoryx_posh/gateway/channel.hpp

```diff
@@ -17,6 +17,8 @@
 
 #include "iceoryx_posh/capro/service_description.hpp"
 #include "iceoryx_posh/iceoryx_posh_types.hpp"
+#include "iceoryx_posh/popo/publisher_options.hpp"
+#include "iceoryx_posh/popo/subscriber_options.hpp"
```

Comment:
Are the includes needed? It's a template parameter down there

Reply:
nope, those are remainders of a failed first attempt. Will remove it in the follow up PR